Project3/index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Heebo:400,500,700,800,900|Mukta+Mahee:400,500,600,700,800"
+        rel="stylesheet">
+    <title>Timeline</title>
+</head>
+
+<body>
+    <h1>A day in Orlando</h1>
+    <figure id="hour1">
+        <h2>1. 6:30am</h2>
+        <a href="#hour1">
+            <img src="images/orlando1.jpg" alt="photo of two bedroom" width="300 " height="300">
+        </a>
+        <figcaption>
+            After arriving in Orlando International Airport, I ate lunch with my travel mates and then went to premium
+            outlet to shop. While we were shopping, we got what we wanted to buy before coming here and used ‘Uber’ app
+            to go to hotel we already booked. We unpacked our room and went to a swimming pool in the hotel. We were not
+            really hungry, so we chose to eat a kid meal that consisted small fries and chicken tenders. I slept earlier
+            than usual, because we needed to get up earlier to watch ‘Harry Potter World’ in the Universal Studio’s
+            famous place.
+            I had to wake up earlier to prepare myself before going to the place, so I washed my face and brushed my
+            teeth. Also, I did my makeup and I dressed up in different costumes. In addition to myself, there were
+            others who spent time styling themselves. We checked to see if we had our passports, hotel key cards and
+            ticket confirmation before we left. We used our hotel shuttle to get to Universals Studios and it took
+            15minutes to reach there.
+
+            <a href="#" class="button-reset">&times</a>
```

Comment:
Since this is an `a` element and not a `button`, the `button-reset` class isn't necessary. This link, however, could use some styles to make it more visible on the dark background colors.
Also, the symbol here needs a semicolon at the end:
```suggestion
            <a href="#" class="button-reset">&times;</a>
```
This is true here and on all the other uses of `&times;`